spec/cb/responses/api_response_spec.rb

```diff
@@ -117,6 +119,15 @@ def extract_models
             expect(DumbValidResponse.new(valid_input_hash).respond_to?(:errors)).to eq true
           end
         end
+        
+        context 'missing required field' do
+          it 'throws ExpectedResponseFieldMissing with specific error message' do
+            expect{DumbValidResponse.new(other_input_hash)}.
+                to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+              expect(ex.message).to eql("Response field missing 'stuff' for Cb::DumbValidResponse")
```

Comment:
`eq` instead of `eql` please, and we prefer no parenthesis on method calls for single parameter on the end of these:

```rb
expect(ex.message).to eq "Response field missing 'stuff' for Cb::DumbValidResponse"
```